Tests/SentryTests/Integrations/SessionReplay/SentrySessionReplayIntegrationTests.swift

```diff
@@ -21,6 +21,18 @@ class SentrySessionReplayIntegrationTests: XCTestCase {
         }
     }
     
+    private class TestCrashWrapper: SentryCrashWrapper {
+        var traced = true
```

Comment:
This will always be set to true by default in the init, and will never be modified otherwise, so we could declare this thusly:
```suggestion
        let traced: Bool
```